lib/utapi/utilities.js

```diff
@@ -8,116 +8,187 @@ import _config from '../Config';
+                    // eslint-disable-next-line no-console
+                    console.log(responseBody);
+                    process.exit(0);
+                } else {
+                    logger.error('request failed with HTTP Status ', {
+                        statusCode: response.statusCode,
+                        body: responseBody,
+                    });
+                    process.exit(1);
+                }
+            });
         });
-    });
-    auth.client.generateV4Headers(request, '', accessKey, secretKey, 's3');
-    if (verbose) {
-        logger.info('request headers', { headers: request._headers });
+        auth.client.generateV4Headers(request, '', accessKey, secretKey, 's3');
+        if (verbose) {
+            logger.info('request headers', { headers: request._headers });
+        }
+        if (this.metric === 'buckets') {
+            const buckets = metricType;
+            request.write(JSON.stringify({ buckets, timeRange }));
+        } else if (this.metric === 'accounts') {
+            const accounts = metricType;
+            request.write(JSON.stringify({ accounts, timeRange }));
+        }
+        request.end();
     }
-    request.write(JSON.stringify({ buckets, timeRange }));
-    request.end();
 }
 
-/**
- * This function is used as a binary to send a request to utapi server
- * to list bucket metrics
- *
- * @return {undefined}
- */
-export function listBucketMetrics() {
-    commander
-        .version('0.0.1')
-        .option('-a, --access-key <accessKey>', 'Access key id')
-        .option('-k, --secret-key <secretKey>', 'Secret access key')
-        .option('-b, --buckets <buckets>', 'Name of bucket(s)' +
-        'with a comma separator if more than one')
-        .option('-s, --start <start>', 'Start of time range')
-        .option('-e --end <end>', 'End of time range')
-        .option('-h, --host <host>', 'Host of the server')
-        .option('-p, --port <port>', 'Port of the server')
-        .option('--ssl', 'Enable ssl')
-        .option('-v, --verbose')
-        .parse(process.argv);
+class ListMetrics {
+    /**
+     * Assign the metric property to an instance of this class
+     * @param {string} metric - The metric type (e.g., 'buckets', 'accounts')
+     */
+    constructor(metric) {
+        this.metric = metric;
+    }
 
-    const { host, port, accessKey, secretKey, start, end,
-        buckets, verbose, ssl } =
-        commander;
-    const requiredOptions = { host, port, accessKey, secretKey, buckets,
-        start };
-    Object.keys(requiredOptions).forEach(option => {
-        if (!requiredOptions[option]) {
-            logger.error(`missing required option: ${option}`);
-            commander.outputHelp();
-            process.exit(1);
-            return;
+    listMetrics() {
+        let metricType;
+        let metricTypeHelp;
+        if (this.metric === 'buckets') {
+            metricType = '-m, --buckets <buckets>';
```

Comment:
Bucket-level metrics was previously `-b`, but with the addition of account-level metrics the obvious `-a` argument is already taken. Thus I propose that the metrics-level should take a universal metric argument like `-m`. Input on what the argument should be for the various metrics, would be helpful here.